pkg/kubelet/apis/config/v1beta1/defaults_linux.go

```diff
@@ -24,4 +24,5 @@ var DefaultEvictionHard = map[string]string{
 	"nodefs.available":  "10%",
 	"nodefs.inodesFree": "5%",
 	"imagefs.available": "15%",
+	"pid.available":     "10%",
```

Comment:
docs will need to be updated: https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/#hard-eviction-thresholds

Reply:
Done. 
Fix doc:https://github.com/kubernetes/website/pull/26255